src/app/components/ImageCard/ImageCard.tsx

```diff
@@ -0,0 +1,65 @@
+import Image from 'next/image';
+import { type CSSProperties, memo } from 'react';
+import styles from './ImageCard.module.scss';
+import classNames from 'classnames';
+
+interface ImageCardProps {
+  imageUrl: string;
+  imageAlt: string;
+  children: React.ReactNode;
+  imageFirst?: boolean;
+  imageWidth?: number;
+  imageHeight?: number;
+  imageStyle?: CSSProperties;
+}
+
+const baseClasses = {
+  container:
+    'grid grid-cols-2 justify-items-center gap-4 xl:justify-items-start xl:gap-14',
```

Comment:
Instead of splitting out classes, what do you think about spitting each of these into their own components? I'm doing something [similar right now in my PR](https://github.com/CDCgov/dibbs-site/pull/59/files#diff-4edbd963b4300eed137680e143e12a95c6476e19a4a1e14b2a8b5a4022a4b50cR8). Makes it really straightforward to tell how each component will impact the layout and lets us encapsulate any logic inside it too.

For example `container` could look like this, 
```ts
interface ContainerProps {
  children: React.ReactNode;
  imageFirst?: boolean;
}

function Container({ children, imageFirst = true }: ContainerProps) {
  let imageFirstClass = 'xl:grid-cols-[1fr_2fr]';
  if (!imageFirst) {
    imageFirstClass = 'xl:grid-cols-[2fr_1fr]';
  }

  return (
    <div
      className={classNames(
        'grid grid-cols-2 justify-items-center gap-4 xl:justify-items-start xl:gap-14',
        imageFirstClass,
        styles['image-card'],
      )}
    >
      {children}
    </div>
  );
}
```